msg/MotorController.msg

```diff
@@ -0,0 +1,6 @@
+string NAME_DEFAULT = "default"
+string NAME_FRONT = "front"
+string NAME_REAR = "rear"
+
+string name
+panther_msgs/MotorControllerState state
```

Comment:
In `MotorControllerState` we refer to "left" and "right" motor runtime errors. How does it correlate with Lynx?